.github/workflows/cd.yaml

```diff
@@ -113,6 +113,23 @@ jobs:
           JRELEASER_PROJECT_VERSION: ${{ steps.extract-version.outputs.version }}
           JRELEASER_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           JRELEASER_HOMEBREW_GITHUB_TOKEN: ${{ steps.generate_homebrew_token.outputs.token }}
+    
+      - name: Generate Scoop token
+        uses: tibdex/github-app-token@v2
+        id: generate_scoop_token
+        with:
+          app_id: ${{ secrets.SCOOP_APP_ID }}
+          private_key: ${{ secrets.SCOOP_APP_PRIVKEY }}
+
+      - name: Publish release to Scoop
```

Comment:
This is not how jreleaser is working. The publish will publish both to Homebrew and scoop. No need to call it twice 

Also you just need one token since we use the same service account/app ID for both. Only the repository is different

Why not just adding 

```
JRELEASER_SCOOP_GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
```

And rename the steps to somethink like `Generate token` instead of `Generate Homebrew token`

Reply:
Sadly those one no. Because they are set at org level

![Screenshot from 2024-12-30 16-35-42](https://github.com/user-attachments/assets/a603ae0e-439a-46bc-9d1c-181254c182cd)